assignment01/Exercise3.java

```diff
@@ -0,0 +1,22 @@
+package assignment01;
+
+public class Exercise3 {
+    public boolean checkPalindrome(String inputString) {
+        boolean band = false;
+        String compare = new StringBuilder(inputString).reverse().toString();
+
+        if (inputString.equals(compare)) {

```

Comment:
You can simply return the result of `inputString.equals(compare)` since the value that the equal method returns is a boolean that can be either false or true